dataflow/gemma-flex-template/requirements-test.txt

```diff
@@ -0,0 +1,7 @@
+google-cloud-aiplatform==1.49.0
+google-cloud-dataflow-client==0.8.10
+google-cloud-pubsub==2.21.4
+google-cloud-storage==2.16.0
+pytest==7.4.0
```

Comment:
Pytest is now up to 8.x. Unless this and other dependencies need to be backpinned, please pin the latest version. Otherwise, as soon as this PR is merged and renovate runs, it will create a PR to bump these anyway.

Reply:
done